microsoft/testsuites/performance/kvmperf.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import inspect
+import time
+from typing import Any, Dict, List
+
+from lisa import (
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    schema,
+    search_space,
+    simple_requirement,
+)
+from lisa.environment import Environment
+from lisa.features import Disk
+from lisa.messages import DiskPerformanceMessage, DiskSetupType, DiskType
+from lisa.node import RemoteNode
+from lisa.tools import Lscpu
+from microsoft.testsuites.nested.common import (
+    connect_nested_vm,
+    parse_nested_image_variables,
+)
+from microsoft.testsuites.performance.common import (
+    format_disk,
+    handle_and_send_back_results,
+    make_raid,
+    run_perf_test,
+    stop_raid,
+)
+
+
+@TestSuiteMetadata(
+    area="storage",
+    category="performance",
+    description="""
+    This test suite is to validate performance of nested VM using FIO tool.
+    """,
+)
+class KVMPerformance(TestSuite):  # noqa
+    TIME_OUT = 12000
+
+    @TestCaseMetadata(
+        description="""
+        This test case is to validate performance of nested VM using fio tool
+        with single l1 data disk attached to the l2 VM.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            disk=schema.DiskOptionSettings(
+                disk_type=schema.DiskType.PremiumSSDLRS,
```

Comment:
What's the reason to limit the disk type in this test?

Reply:
This is the default disk type used in all the performance testcases written so far